### ISSUE TITLE
When running Approval tests, there is NameError: name 'version_number' is not defined

### ISSUE THREAD

**Issue by aaltat** [approvals/ApprovalTests.Python#103]
When using Approval tests 3.1.1, I see error when running tests 
```
"/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/approvaltests/__init__.py", line 8, in <module>
    __version__ = version_number
NameError: name 'version_number' is not defined
Error: Process completed with exit code 1.
```

Error did not happen with version 3.1.0.

Full stack trace at https://github.com/MarketSquare/robotframework-browser/runs/4304274907?check_suite_focus=true

**Comment by aaltat** [comment]
I can reproduce the problem with this
```
Python 3.10.0 (v3.10.0:b494f5935c, Oct  4 2021, 14:59:19) [Clang 12.0.5 (clang-1205.0.22.11)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import approvaltests
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/user/workspace/tmp/.venv_test/lib/python3.10/site-packages/approvaltests/__init__.py", line 8, in <module>
    __version__ = version_number
NameError: name 'version_number' is not defined
>>> 

```

I only have approval test installed in Python virtual enviroment and my pip list looks like:
```

**Comment by aaltat** [comment]
When I download version 3.1.1 ( https://pypi.org/project/approvaltests/3.1.1/#files ) from Pypi and look at the [version.py](https://github.com/approvals/ApprovalTests.Python/blob/master/approvaltests/version.py) I see
```
version="v3.1.1"
```
But [__init__.py](https://github.com/approvals/ApprovalTests.Python/blob/master/approvaltests/__init__.py) uses:
```
__version__ = version_number
```
 I think the bug is in [GitHub action](https://github.com/approvals/ApprovalTests.Python/blob/master/.github/workflows/publish.yml#L30). I think `echo version` should be `echo version_number`.

**Comment by aaltat** [comment]
Because of this bug, it breaks example the https://pypi.org/project/pytest-approvaltests/

**Comment by claremacrae** [comment]
> +1: for the pkgsrc package, I get the same NameError as @aaltat when trying to run the pytest example from the README. And patching `approvaltests/version.py` to say `version_number` instead of `version` makes things work.

Best to ping Llewellyn directly, as AFAIK he doesn't see/receive GitHub notifications...

There is a PR for this already, I see...

**Comment by aaltat** [comment]
Would someone be able to ping him, I have no idea where to reach him?

**Comment by claremacrae** [comment]
I've just pinged Llewellyn :-)

A small group works on it most Sundays, so it may get picked up then...

**Comment by schmonz** [comment]
3.3.2 solves this for me. If @aaltat agrees, this can be closed.

**Comment by isidore** [comment]
Thanks, fixed in 3.3.2 
Sorry for the bug
